.github/workflows/release.yml

```diff
@@ -24,11 +24,19 @@ jobs:
         run: |
```

Comment:
Can you also add these same changes to the `compile.yml` workflow?

Reply:
I just did, let's see if the CI works ;)